SECP-service/src/main/resources/assets/app/views/profile/user-profile.html

```diff
@@ -0,0 +1,44 @@
+<div>
+    <div>
+        <div id="editbutton" ng-show="EditProfileAvailable()">
+            <input type="button" ng-click="EditProfile()" value="Edit Profile" />
+        </div>
+    </div>
+    <div id="profile">
+        <div>
+            <img src="https://scontent.fykz1-1.fna.fbcdn.net/v/t1.0-1/c90.210.540.540/s160x160/19510125_10213767937303246_5095195573619394298_n.jpg?oh=7768b901dcef45df1ccfa57878be3e89&oe=5AC0FE69"
+                class="img-responsive img-circle">
+        </div>
+        <input type="file" id="photo-input" ng-show="IsImageFileVisible" />
+        <input type="text" name="display-name" id="display-name" ng-disabled="IsDisplayNameReadOnly" value="Lava Tahir" />
+        <p id="work-email">lavatahir@gmail.com</p>
+        <p id="username">ltahir</p>
+
+        <div id="user-security-settings">
+            <h3 class="list-heading " id="permission-heading ">Permission Level</h3>
+            <ul class="list-group " id="permission-item ">
+                <li class="list-group-item " id="permission-heading ">Top Secret</li>
+            </ul>
+            <h3 class="list-heading " id="role-heading ">Role(s)</h3>
+            <ul class="list-group " id="role-items ">
+                <li class="list-group-item ">Security</li>
```

Comment:
I could only see four roles when i ran your branch. It's not showing the other ones. Please address this.

Reply:
? This is very weird. This is missing a lot of roles I made to test the scrollbar. I don't know why... sourcetree says my changes have been pushed to this branch. Even when I check it says it has been commited (commit no: be988e0)

Reply:
it FINALLY shows the latest commits. Github making me look crazy